containers/proxy-squid-image/squid_image.conf

```diff
@@ -0,0 +1 @@
+# squid_image.conf
```

Comment:
Couldn't you add some useful comment explaining how to add squid config? That would surely be better than an empty file. And the filename could be something like `squid_example.conf` or so.